src/formatters/index.js

```diff
@@ -46,8 +46,23 @@ export function generateStats(data: Array<CheckstyleItem>): Stats {
 }
 
 export function formatOutput(format: string, data: Array<CheckstyleItem>) {
-  if (!{}.hasOwnProperty.call(formatters, format)) {
-    throw new Error(`Formatter with name '${format}' does not exist.`)
+  let formatter
+  if (/^require:/.test(format)) {
```

Comment:
You would only call `require` if the name is not one of the builtins, so it shouldn't matter that it is slow. But sure, being explicit is fine.